src/gccrs/mod.rs

```diff
@@ -61,8 +71,7 @@ impl Gccrs {
         }
     }
 
-    /// Convert arguments given to `rustc` into valid arguments for `gccrs`
-    pub fn handle_rust_args() -> Result {
+    fn cfg_print() -> Result {
```

Comment:
Can you add a fixme that the output needs to be adapted for the target triple?

Reply:
Good catch, thanks. Done in 98688f7